src/outlineView.ts

```diff
@@ -187,6 +203,23 @@ function hasEqualContent(ot1: OutlineTree[], ot2: OutlineTree[]) {
   return true
 }
 
+function makeOutlineToolbar() {
+  const toolbar = document.createElement("span")
+  toolbar.className = "outline-toolbar"
+
+  const revealCursorButton = document.createElement("button")
+  revealCursorButton.innerHTML = "Reveal Cursor"
+  revealCursorButton.className = "btn outline-btn"
+
+  revealCursorButton.addEventListener("click", () => {
+    // @ts-ignore: internal API
+    atom.commands.dispatch(atom.workspace.getElement(), "outline:reveal-cursor")
```

Comment:
```suggestion
    atom.commands.dispatch(atom.views.getView(atom.workspace), "outline:reveal-cursor")
```
then you won't need ts-ignore.

Reply:
That would work for now, but I am going to eventually add this function to the @types/atom.

Reply:
https://github.com/atom-community/atom-ide-outline/pull/98/commits/e43475846c9e029747ab183ff26d9869fd810f19